docs/api.json

```diff
@@ -478,7 +478,7 @@
             {
                 "is_required": false, 
                 "type": "str", 
-                "name": "file_name<file_name>", 
+                "name": "file_name", 
```

Comment:
The angle brackets need to be removed so my API Code generation can work properly